Unable to install covfee on MacOS and Ubuntu

`pip install -e .`

returns 

```
e/include -fPIC -DHAVE_SYS_UN_H=1 -Ibundled/zeromq/include -Izmq/utils -I/home/soren/miniconda3/envs/covfee/include/python3.11 -c zmq/backend/cython/_device.c -o build/temp.linux-x86_64-cpython-311/zmq/backend/cython/_device.o
      zmq/backend/cython/_device.c:217:12: fatal error: longintrepr.h: No such file or directory
        217 |   #include "longintrepr.h"
            |            ^~~~~~~~~~~~~~~
      compilation terminated.
      ************************************************
      ************************************************
      ************************************************
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyzmq
Successfully built covfee flask-jwt-extended halo MarkupSafe
Failed to build pyzmq
ERROR: Could not build wheels for pyzmq, which is required to install pyproject.toml-based projects

```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,3 +1,6 @@
+[project]
+requires-python = ">3.7,<3.11"
+
 [build-system]
 requires = [
     "setuptools>=42",

